src/components/measureLanding/measureList/actionCenter/draftAction/DraftAction.tsx

```diff
@@ -56,8 +56,8 @@ export default function DraftAction(props: PropTypes) {
 
   const draftLookup = useCallback(
     async (measureList: Measure[]) => {
-      const measureSetList = measureList.map((m) => m.measureSetId);
-      if (measureList.length > 0) {
+      const measureSetList = measureList?.map((m) => m.measureSetId);
+      if (measureList?.length > 0) {
```

Comment:
try using lodash: !_.isEmpty(measureList) it works for if measureList is undefined and also if there is empty list